back/engines/commercial/idea_custom_fields/app/controllers/idea_custom_fields/web_api/v1/admin/idea_custom_fields_controller.rb

```diff
@@ -202,7 +201,6 @@ def update_option!(option, option_params, errors, field_index, option_index)
     end
 
     def delete_option!(option)
-      SideFxCustomFieldOptionService.new.before_destroy option, current_user
```

Comment:
A main bugfix (of 2).

Since `before_destroy` only calls `UserCustomFieldService.new.delete_custom_field_option_values`, which is only concerned with removing User.custom_field_values (either just the value, or both the key & value)), we now don't call it.